project/README.md

```diff
@@ -0,0 +1,54 @@
+# KeyChain
+
+## Table of contents
+
+- [Introduction](#introduction)
+- [Instructions](#instructions)
+  - [Group formation](#group-formation)
+- [Evaluation](#evaluation)
+
+---
+
+## Introduction
+
+The main objective of this project is to implement your own decentralized Blockchain [key-value store](https://en.wikipedia.org/wiki/Key-value_database).
+
+---
+
+## Instructions
+
+### Group formation
+
+Please register your group at the [Montefiore submission system](https://submit.montefiore.ulg.ac.be/), also if you are working alone.
+
+---
+
+## Evaluation
+
+This project is a requirement to pass INFO8002, and **must** be completed and submitted with all deliverables to the [Montefiore submission system](https://submit.montefiore.ulg.ac.be/) by the **hard deadline** of `21/12/2018 23:59`. You are allowed to implement the project by yourself, or in groups with a maximum of *3* students (see [group formation](#group-formation)).
```

Comment:
Merge 'Group formation' and 'Evaluation'? (and put everything under Instructions)